### PR TITLE
C4lyp5o

### PR DESCRIPTION
**Pull request by c4lyp5o**

### CODE REVIEW

**Comment by c4lyp5o** [comment]
@nocturnalmc mohon pengecekkan nya tuan yg mulia

**Review by nocturnalmc — CHANGES_REQUESTED** [review]
Move to env

File: controllers/adminAPI.js [hunk 1]
```diff
@@ -371,6 +293,20 @@ const loginUser = async (req, res) => {
     });
   }
   // if admin
+  // PERLIS TEST DEV ANY PASS CAN LOGIN
+  if (
+    process.env.BUILD_ENV === 'dev' &&
+    (adminUser.negeri === 'Perlis' || adminUser.daerah === 'Arau')
```
Comment (nocturnalmc) [inline controllers/adminAPI.js:302]: to env

Comment (c4lyp5o) [reply controllers/adminAPI.js]: fixed in [fix: buang kod yang tidak 'diperlukan'](https://github.com/nocturnalmc/gp2/pull/4358/commits/cabc4ee9599ea120ab7e6352074f1073ec4f732c)

**Review by nocturnalmc — APPROVED** [review]

**Pull request merged**